src/Microsoft.ML.Tokenizers/Model/BPE.cs

```diff
@@ -177,7 +177,7 @@ private Bpe(Stream vocabStream, Stream? mergesStream, string? unknownToken, stri
         /// Encode a text string to a list of tokens.
         /// </summary>
         /// <param name="text">The text to encode.</param>
-        /// <param name="isSpecialToken">Indicate if the token is a special token.</param>
+        /// <param name="isSpecialToken">Indicate if the text is a special token.</param>
```

Comment:
From the implementation, the parameter has zero effect. Might want to call that out in the XML docs, too.